CHANGES.md

```diff
@@ -6,6 +6,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### Fixed and improved
 
+* Changed `iam-database-backup` to exit early if ZooKeeper is not running. (DCOS_OSS-5353)
```

Comment:
It's the other way round, no?

@timaa2k

Reply:
You're right, oops!

Reply:
Also, we're not changing `iam-database-backup` here. The program is called differently.